docs-2.0/5.configurations-and-logs/1.configurations/1.configurations.md

```diff
@@ -83,10 +85,37 @@ Nebula Graph 为每个服务都提供了两份初始配置文件`<service_name>.
 !!! caution
     不建议修改`local_config`的值为`false`。修改后 Nebula Graph 服务启动后会先尝试从 Meta 服务获取缓存的配置信息，可能导致集群配置不一致，造成未知风险。
 
+### Docker Compose 集群的配置文件
+
+对于使用 Docker Compose 创建的集群，集群的配置文件的默认路径为`{home}/nebula-docker-compose/docker-compose.yaml`。配置文件中的`command`下面的参数为各服务的启动参数。
+
+### Nebula Operator 集群的配置文件
+
+对于通过 Nebula Operator 使用 Kubectl 方式创建的集群，集群的配置文件的路径为用户创建集群 YAML 文件的路径。用户可通过配置文件中的`spec.{graphd|storaged|metad}.config`参数修改集群各个服务的相关配置。
+
+!!! note
+
+    通过 Helm 部署的集群，暂不支持修改集群服务的相关配置。
+
+关于 Nebula Graph 的配置相关的更多信息，参见 [Nebula Config](https://siwei.io/sketches/nebula-config-explained/)。
+
 ## 修改配置
 
-默认情况下，所有 Nebula Graph 服务从配置文件获取配置。用户可以按照以下步骤修改配置并使其生效：
+默认情况下，所有 Nebula Graph 服务从配置文件获取配置。用户可以按照以下步骤修改配置并使其生效。
+
+* 针对使用源码、RPM/DEB、TAR 包安装的集群：
+
+  1. 使用文本编辑器修改目标服务的配置文件并保存。
+
+  2. 选择合适的时间重启**所有** Nebula Graph 服务使修改生效。
 
-1. 使用文本编辑器修改目标服务的配置文件并保存。
+* 针对使用 Docker Compose 安装的集群：
+  
+  1. 在目录`nebula-docker-compose`内执行命令`docker-compose down`停止集群服务。
+  2. 在文件`{home}/nebula-docker-compose/docker-compose.yaml`中，修改服务配置。
+  3. 在目录`nebula-docker-compose`内执行命令`docker-compose pull`。
```

Comment:
For purely changing command field in YAML file, no docker compose down or pull or is required. Another `docker-compose up -d` will do the job.

ref: https://stackoverflow.com/a/54114314/1402404
> example on changing container2 configuration via the yaml file:

```
> docker-compose up -d
Starting container1 ... done

# change container2
> docker-compose up -d
container1 is up-to-date
Creating container2 ... done
```